README.md

```diff
@@ -3,5 +3,11 @@ SymfonyMakerBundle
 
 Symfony Maker helps you creating empty commands, controllers, form classes,
 tests and more so you can forget about the required boilerplate code. This
-bundle replaces [SensioGeneratorBundle](https://github.com/sensiolabs/SensioGeneratorBundle)
-in modern Symfony applications.
+bundle replaces [SensioGeneratorBundle][1] in modern Symfony applications and
```

Comment:
I'm sure we can claim that it replaces the generator bundle. At least, not yet as the main feature is CRUD generation, which this bundle does not do yet.